CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 This is the changelog of Grafeas server releases. For more information on
 versionining, see [versioning](docs/versioning.md) document.
 
+v0.1.5:
+  * Upgraded to golang 1.14.0
+  * Addeded last_scan_time to discovery occurrences
+  * Added support for Windows updates
+  * Changed resource type API Group to top level domain
+  * Added API config annotations
+  * Removed UNKNOWN types
```

Comment:
Removed from where?

Reply:
I think I got this wrong. I. was looking at the commit history which refers to a very old PR #3. 
https://github.com/grafeas/grafeas/commit/8ab5b883308fe4bd3d4862b9d2d10c91a346c8d9

Reply:
ping on this one -- do you have a related PR handy?